analyzer/codechecker_analyzer/cmd/analyze.py

```diff
@@ -608,17 +608,29 @@ def add_arguments_to_parser(parser):
                                metavar='checker/group/profile',
                                default=argparse.SUPPRESS,
                                action=OrderedCheckersAction,
-                               help="Set a checker (or checker group) "
-                                    "to BE USED in the analysis.")
+                               help="Set a checker (or checker group), "
+                                    "profile or guideline "
+                                    "to BE USED in the analysis. In case of "
+                                    "ambiguity the priority order is profile, "
+                                    "guideline, checker name (e.g. security "
+                                    "means the profile, not the checker "
+                                    "group). 'profile:' and 'guideline:' "
+                                    "prefixes can be used.")
```

Comment:
Some example would be great too:
```suggestion
                                    "prefixes can be used. For example: 'profile:security', 'guideline:sei-cert'.")
```
Please update the help message of the `check` command and in the user guide too.